SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/TetrahedronFEMForceField.h

```diff
@@ -163,6 +163,7 @@ class TetrahedronFEMForceField : public core::behavior::ForceField<DataTypes>, p
     friend class TetrahedronFEMForceFieldInternalData<DataTypes>;
 
     Real m_restVolume;
+    sofa::helper::ColorMap* m_VonMisesColorMap;
```

Comment:
why a pointer ? (curious)

Reply:
why not?

Reply:
it is only created if we draw things... I don't have a better answer.